src/ontology/cl-edit.owl

```diff
@@ -5721,9 +5722,13 @@ SubClassOf(obo:CL_0000238 obo:CL_0000240)
 
 # Class: obo:CL_0000239 (brush border epithelial cell)
 
+AnnotationAssertion(obo:IAO_0000115 obo:CL_0000239 "An epithelial cell characterized by the presence of a brush border on its apical surface, which increases the surface area for absorption. These cells are predominantly found lining the small intestine and the proximal tubules of the kidneys.")
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000239 <https://doi.org/10.1152/ajpgi.00005.2011>)
+AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000239 "PMID:25422372")
+AnnotationAssertion(rdfs:comment obo:CL_0000239 "All brush border cells are columnar-cuboidal.  The formal logical definition of this term asserts this.")
```

Comment:
> The formal logical definition of this term asserts this.

Consider removing this sentence.

Reply:
I understand why you'd want to remove.  It's my attempt at leaving a node to explain the hidden GCI pattern in the definition:

EquivalentTo: 'epithelial cell' and has_part some 'brush border'
SubClassOf: columnar/cuboidal epithelial cell

This pattern is justified by that all brush border epiethlial cells are columnar cuboidal.  Use of the pattern means that ontology editors don't need to remember to classify as columnar cuboidal in order to get automated classification.  However because hidden GCIs* are powerful and their effects can only be traced using reasoner explanations, I think they need to be used sparingly and with text that supports/explains them.

I think in this case we should just move the assertion ("all brush border epithelial cells are columnar cuboidal) into the definition.   It is also possible to leave a comment on the subClassOf axiom directly, although that would be non-standard.


\* This is a hidden GCI because it could be rewritten as the GCI:
  'epithelial cell' and has_part some 'brush border' subClassOf columnar/cuboidal epithelial cell